daemon/containerd/image_list.go

```diff
@@ -254,8 +255,9 @@ type imageFilterFunc func(image images.Image) bool
 // setupFilters constructs an imageFilterFunc from the given imageFilters.
 //
 // TODO(thaJeztah): reimplement filters using containerd filters: see https://github.com/moby/moby/issues/43845
-func (i *ImageService) setupFilters(ctx context.Context, imageFilters filters.Args) (imageFilterFunc, error) {
+func (i *ImageService) setupFilters(ctx context.Context, imageFilters filters.Args) ([]string, imageFilterFunc, error) {
```

Comment:
Not a blocker (as it's not exported), but we could consider to use either named return variables (to "document" what they are), or update the GoDoc of the function to describe what it returns.

Reply:
Changed parameters to be named and described them in Godoc.